### ISSUE TITLE
Will there every be an automatic installer for the mod

### ISSUE THREAD

**Issue by Konclan** [BEEmod/BEE2.4#279]
There are some people who are confused on the installation and I think it would be easier if we can just download something that downloads both packages automatically. Also an EXE shortcut I think would be better instead of having it hidden in the bin folder.

**Comment by LautaroL20** [comment]
Thats a very good suggestion for the mod.

**Comment by LautaroL20** [comment]
Yes, but not until the mod reaches a stage were the known bugs are all solved.

**Comment by calclover2514** [comment]
An installer can be found [here](https://github.com/calclover2514/BEE2.4-Installer-Automatic/releases).
(Issue #1121)

**Comment by LautaroL20** [comment]
Interesting, i will take a look at it and will share your contribution with the community if its fine.

**Comment by LautaroL20** [comment]
Ive shared your installed with the Beemod community at the Discord server, here: https://discordapp.com/invite/hnGFJrz
